src/frontends/tensorflow/tests/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ if (NOT ENABLE_INTEL_CPU)
     set(EXCLUDED_TESTS ${CMAKE_CURRENT_SOURCE_DIR}/compilation.cpp)
 endif()
 
+ov_check_pip_packages(REQUIREMENTS_FILE "${CMAKE_CURRENT_SOURCE_DIR}/requirements.txt"
+                      MESSAGE_MODE WARNING
+                      WARNING_MESSAGE "TensorFlow testing models weren't generated, some tests will fail due models not found"
+                      RESULT_VAR tensorflow_FOUND)
+
+if(tensorflow_FOUND)
+    set(ctest_labels LABELS OV UNIT)
```

Comment:
Currently, some GHA pipelines use ctest to run tests via labels:
https://github.com/openvinotoolkit/openvino/blob/44ac9099b947af4dfd9c50067036aa558a0643f6/.github/workflows/linux_conditional_compilation.yml#L156
So, you can mark tests with specific labels to unite them into groups and them run using a single command.

Current PR add `UNIT` label (which is used in CI) only for cases when models are converted, otherwise CI will run those tests and they fail